api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -133,4 +133,18 @@
      * @return all currently registered {@link ConfigSource ConfigSources} sorted by descending ordinal and ConfigSource name
      */
     Iterable<ConfigSource> getConfigSources();
+    
+    /**
+     * Convert an abritrary value to the specified type using the converters known to this instance
+     * 
+     * @param <T>
+     *             the conversion type
+     * @param value
+     *             the value to convert
+     * @param type
+     *             the type into which the value should be converted
+     * @return the converted value
+     * @throws java.lang.IllegalArgumentException if the value cannot be converted to the specified type.
+     */
+    <T> T convert(String value, Class<T> type);
```

Comment:
I don't think this functionality should belong to Config. See my [comments](https://github.com/eclipse/microprofile-config/issues/492#issuecomment-575568151)

Reply:
See also [my disagreement](https://github.com/eclipse/microprofile-config/issues/492#issuecomment-575615818).  The change should be accepted.